packages/patternfly-3/patternfly-react/src/components/DateTimePicker/TimeComponents/PickTimeClock.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop } from '../../../common/helpers';
+
+class PickTimeClock extends React.Component {
+  state = {
+    ampm: this.props.time.getHours() >= 12 ? 'PM' : 'AM'
+  };
+  static getDerivedStateFromProps(nextProps, prevState) {
+    return {
+      ampm: nextProps.time.getHours() >= 12 ? 'PM' : 'AM'
+    };
+  }
+  setTime = (type, amount) => {
+    const { time } = this.props;
+    if (type === 'H') {
+      time.setHours(time.getHours() + amount);
+    } else if (type === 'M') {
+      time.setMinutes(time.getMinutes() + amount);
+    }
+    this.props.setSelected(time);
+  };
+  toggleAMPM = () => {
+    const { time } = this.props;
+    if (this.state.ampm === 'AM') {
+      time.setHours(time.getHours() + 12);
+      this.setState({ ampm: 'PM' });
+    } else {
+      time.setHours(time.getHours() - 12);
+      this.setState({ ampm: 'AM' });
+    }
+    this.props.setSelected(time);
+  };
+  render() {
+    const { time, toggleTimeTable } = this.props;
+    const minutes = time.getMinutes();
+    const hours = time.getHours() % 12 || 12;
+
+    return (
+      <div className="timepicker-picker">
+        <table>
+          <tr>
```

Comment:
Please add a `tbody` here as the browser throws an error about it:
![selection_024](https://user-images.githubusercontent.com/26363699/53696402-4106cc80-3dcf-11e9-910f-b9502cdde4aa.png)

Reply:
Insert `tbody` will break patternfly css 
But according to [react](https://github.com/facebook/react/issues/5652)  it is  important to have this `tbody`

how about moving the fixed css file to patternfly-react until `patternfly` will fix it in patternfly-core?

Reply:
added tbody